### PR TITLE
Translating - React without ES6

### PR DESCRIPTION
**Pull request by aladin002dz**
working on the page: "React without ES6"

### CODE REVIEW

File: content/docs/react-without-es6.md [hunk 1]
```diff
@@ -159,27 +159,27 @@ class SayHello extends React.Component {
...
 
-* Bind methods in the constructor.
-* Use arrow functions, e.g. `onClick={(e) => this.handleClick(e)}`.
-* Keep using `createReactClass`.
+* ربط التوابع في الدالة البانية..
+* استخدام الدوال السهمية، مثل `onClick={(e) => this.handleClick(e)}`.
+* الاستمرار في استخدام `createReactClass`.
 
-## Mixins {#mixins}
+## المخاليط {#mixins}
```
Comment (Fcmam5) [inline content/docs/react-without-es6.md]: I suggest keeping the original word `mixins` alongside the translation

```suggestion
## المخاليط (Mixins) {#mixins}
```

File: content/docs/react-without-es6.md [hunk 2]
```diff
@@ -222,4 +222,4 @@ ReactDOM.render(
 );
 ```
 
-If a component is using multiple mixins and several mixins define the same lifecycle method (i.e. several mixins want to do some cleanup when the component is destroyed), all of the lifecycle methods are guaranteed to be called. Methods defined on mixins run in the order mixins were listed, followed by a method call on the component.
+إذا كان المُكوّن يستخدم مخاليط متعددة تُعرِّف نفس توابع دورة الحياة (أي مثلًا تريد كل هذه المخاليط إيقاف التابع setInterval()‎ عند تدمير المُكوّن)، فسنضمن استدعاء كافة توابع دورة الحياة. تعمل التوابع المُعرَّفة في المخاليط بنفس الترتيب الذي أوردنا فيه هذه المخاليط متبوعةً باستدعاء التابع على المُكوّن.
```
Comment (Fcmam5) [inline content/docs/react-without-es6.md]: The original text in parentheses was:

```
.. (i.e. several mixins want to do some cleanup when the component is destroyed) ...
```

Is it okay to replace it by the example below?

<div dir="rtl">
<code>
(أي مثلًا تريد كل هذه المخاليط إيقاف التابع setInterval()‎ عند تدمير المُكوّن)
</code>
</div>

**Comment by iRayan7** [comment]
🎉

**Comment by aladin002dz** [comment]
> Thank you Aladin for your contribution and for all what you're doing for the React community!
> I have some suggestions if you don't mind 😄

sure, my pleasure! anything about sharing, learning, and building. I'm in!

**Pull request merged**